config/initializers/fast_blank.rb

```diff
@@ -1,5 +1,5 @@
 if String.method_defined?(:blank_as?)
   class String
-    alias_method :blank?, :blank_as?
+    alias blank? blank_as?
```

Comment:
Any specific reason to prefer the latter?

Reply:
Yes! [here's](https://deepsource.io/gh/withshubh/mastodon/issue/RB-ST1002/description) the detailed description that why we should use the latter